ruby_event_store/lib/ruby_event_store/client.rb

```diff
@@ -1,15 +1,19 @@
 require 'concurrent'
 
 module RubyEventStore
+  DEFAULT_DISPATCHER = PubSub::Dispatcher.new
+
   class Client
     def initialize(repository:,
                    mapper: Mappers::Default.new,
-                   event_broker:  PubSub::Broker.new,
+                   event_broker: PubSub::Broker.new,
```

Comment:
this is always a new, separate instance, initialized for every new client. no shared dependencies.